em_workflows/brt/flow.py

```diff
@@ -503,7 +503,7 @@ def gen_ng_metadata(fp_in: FilePath) -> Dict:
     file_path = fp_in
     asset_fp = Path(f"{file_path.assets_dir}/{file_path.base}.zarr")
 
-    first_zarr_arr = Path(asset_fp.as_posix() + "/0")
+    first_zarr_arr = Path(asset_fp.as_posix() + "/0/0")
 
```

Comment:
This whole block could be updated to use the ZarrImages interface which will take care of this details.

Reply:
Noted!